### PR TITLE
Add OpenShift templates for deployment

### PR DESCRIPTION
**Pull request by fridex**

### CODE REVIEW

File: openshift/buildConfig-template.yaml [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+labels:
+  template: adviser-buildconfig
+  thoth: 0.1.0
+metadata:
+  name: adviser-buildconfig
+  annotations:
+    description: This is Thoth Adviser BuildConfig, this template is meant to be used by Jenkins, but could also be used by humans...
+    openshift.io/display-name: Thoth Adviser BuildConfig
+    version: 0.3.0
+    tags: poc,thoth,ai-stacks,graph-sync-job
```
Comment (goern) [inline openshift/buildConfig-template.yaml]: s/graph-sync-job/adviser/

File: openshift/buildConfig-template.yaml [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+  thoth: 0.1.0
+metadata:
+  name: adviser-buildconfig
+  annotations:
+    description: >
+      This is Thoth Adviser BuildConfig, this template is meant to be used by Jenkins, but could also be
+      used by humans...
+    openshift.io/display-name: Thoth Adviser BuildConfig
+    version: 0.3.0
+    tags: poc,thoth,ai-stacks,graph-sync-job
```
Comment (goern) [inline openshift/buildConfig-template.yaml]: s/graph-sync-job/adviser/

File: openshift/imageStream-template.yaml [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+  thoth: 0.1.0
+metadata:
+  name: adviser-imagestream
+  annotations:
+    description: >
+      This is Thoth Adviser ImageStream, this template is meant to be used by Jenkins, but could also
+      be used by humans
+    openshift.io/display-name: Thoth Adviser ImageStream
+    version: 0.2.0
+    tags: poc,thoth,ai-stacks,graph-sync-job
```
Comment (goern) [inline openshift/imageStream-template.yaml]: adviser

File: openshift/pod-template.yaml [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+labels:
+  template: adviser-pod
+  thoth: 0.1.0
+metadata:
+  name: adviser
+  annotations:
+    description: This is Thoth Core - Adviser
+    openshift.io/display-name: 'Thoth: Adviser'
+    version: 0.3.0
+    tags: poc,thoth,ai-stacks,graph-sync-job
```
Comment (goern) [inline openshift/pod-template.yaml]: adviser

File: openshift/imageStream-template.yaml [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+    template.openshift.io/documentation-url: https://github.com/AICoE/Thoth/
```
Comment (goern) [inline openshift/imageStream-template.yaml]: url

File: openshift/buildConfig-template.yaml [hunk 6]
```diff
@@ -0,0 +1,80 @@
...
+    template.openshift.io/documentation-url: https://github.com/AICoE/Thoth/
```
Comment (goern) [inline openshift/buildConfig-template.yaml]: url

File: openshift/pod-template.yaml [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+             tcpSocket:
+               port: 80
+             initialDelaySeconds: 600
+             failureThreshold: 1
+             periodSeconds: 10
+          env:
+            - name: THOTH_LOG_ADVISER
+              value: 'INFO'
+            # These are intentionally empty - they will be provided on pod run request.
+            - name: THOTH_ADVISER_OUTPUT
```
Comment (goern) [inline openshift/pod-template.yaml]: all ENV are empty strings?!

Comment (fridex) [reply openshift/pod-template.yaml]: I wanted to have them present explicitly - they will be supplied by user-api based on user input (see comment).

**Pull request merged**